tests/test_nacl_loss.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import unittest
+
+import numpy as np
+import torch
+from parameterized import parameterized
+
+from monai.losses import NACLLoss
+
+inputs = torch.tensor(
+    [
+        [
+            [
+                [0.1498, 0.1158, 0.3996, 0.3730],
+                [0.2155, 0.1585, 0.8541, 0.8579],
+                [0.6640, 0.2424, 0.0774, 0.0324],
+                [0.0580, 0.2180, 0.3447, 0.8722],
+            ],
+            [
+                [0.3908, 0.9366, 0.1779, 0.1003],
+                [0.9630, 0.6118, 0.4405, 0.7916],
+                [0.5782, 0.9515, 0.4088, 0.3946],
+                [0.7860, 0.3910, 0.0324, 0.9568],
+            ],
+            [
+                [0.0759, 0.0238, 0.5570, 0.1691],
+                [0.2703, 0.7722, 0.1611, 0.6431],
+                [0.8051, 0.6596, 0.4121, 0.1125],
+                [0.5283, 0.6746, 0.5528, 0.7913],
+            ],
+        ]
+    ]
+)
+targets = torch.tensor([[[1, 1, 1, 1], [1, 1, 1, 0], [0, 0, 0, 0], [0, 0, 0, 0]]])
+
+TEST_CASES = [
+    [{"classes": 3, "dim": 2}, {"inputs": inputs, "targets": targets}, 1.1820],
```

Comment:
Could you please include more tests for other supported kernel_ops and arguments?